packages/redux-dynamic-modules-react/src/DynamicModuleLoader.tsx

```diff
@@ -12,6 +12,9 @@ export interface IDynamicModuleLoaderProps {
     /** Modules that need to be dynamically registerd */
     modules: IModuleTuple;
 
+    // boolean flag to determine if dynamically loaded modules should be automatically unloaded when component is dismounted
+    keepModulesOnUnmount: boolean | undefined;

```

Comment:
Please follow the same pattern as strictMode i.e.  keepModulesOnUnmount.
This is just for consistency.